packages/kbn-apm-config-loader/src/init_apm.ts

```diff
@@ -35,5 +35,5 @@ export const initApm = (
     return payload;
   });
 
-  apm.start(apmConfig);
+  return apm.start(apmConfig);
```

Comment:
Wonder if there's a way to achieve what this PR aims to do without changing this file, any suggestions welcome.

Reply:
@elastic/apm-ui any thoughts on this and the transactions below?

For CLI metrics (on operations) we tend to use the `CiStatsReporter` [example](https://github.com/elastic/kibana/blob/main/kbn_pm/src/cli.mjs#L100), so I'm not especially versed on best practices here.

Reply:
Not an APM expert, but, AFAIK, APM is a singleton client. You can call `initApm` from this package, and import `apm` from the package `elastic-apm-node`, and it should work.

Reply:
Thanks for your hints, I was able to push an update that works without changing this file in [607105c](https://github.com/elastic/kibana/pull/157129/commits/607105c76e1e679900cce3ed7b5b1c1f7cafaf11).